### PR TITLE
MOTS-603: Add ability to filter and sort reports

### PR DESCRIPTION
**Pull request by jkondrat**

### CODE REVIEW

File: mobile/src/utils/sort.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  }
+  return 0;
+};
+
+export const createSorter = (...args) => {
+  return (A, B) => {
+    let ret = 0;
+
+    args.some((sorter) => {
+      const { property, direction = 1 } = sorter;
```
Comment (awalkowiak) [inline mobile/src/utils/sort.js]: Should not this be string asc or desc? What if no direction will be in sorter object? Then we will pass direction = 1 to doSort.

File: mobile/src/utils/sort.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      ret = value;
+
+      return true;
+    });
+
+    return ret;
+  };
+};
+
+export default { createSorter };
```
Comment (awalkowiak) [inline mobile/src/utils/sort.js]: Is this required? You already have named export on createSorter

Comment (jkondrat) [reply mobile/src/utils/sort.js]: There is an eslint rule that enforces us to use default exports: https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/prefer-default-export.md

File: mobile/src/utils/sort.js (hunk 1 above)
Comment (jkondrat) [reply mobile/src/utils/sort.js]: done

File: mobile/src/utils/sort.js (hunk 2 above)
Comment (awalkowiak) [reply mobile/src/utils/sort.js]: Ok, then no need for named export and export default without destructuring should be enough (if we have only one function exported)

**Pull request merged**